app/grandchallenge/profiles/forms.py

```diff
@@ -50,6 +51,13 @@ def save(self, *args, **kwargs):
 
         return instance
 
+    def clean(self):
+        cleaned_data = super().clean()
+        url = cleaned_data.get("website")
+
+        if not url.startswith("https"):
```

Comment:
```suggestion
        if not url.startswith("https://"):
```

If we don't include the `://` here then the same linking problem will still exist. Also add this to the command on L13.